### ISSUE TITLE
[studio] Allow specifying a ${siteName} placeholder in the blob-store-config.xml 

### ISSUE THREAD

**Issue by avasquez614** [craftercms/craftercms#5277]
#### Is your feature request related to a problem? Please describe.
Is very common to have a blob store bucket per site in a Crafter environment with multiple sites. When the sites use the same remote things can get messy since you need to specify the bucket in the `blob-store-config.xml` with the hardcoded site name and make sure this config is not pushed to the remote. 

#### Describe the solution you'd like
Allow specifying a site name placeholder in `blob-store-config.xml`, like `${siteName}` or `${crafter:siteName}`, which is automatically replaced by the configuration reader.

It would be good to have this too in all our configuration files

#### Describe alternatives you've considered
N/A

**Comment by sumerjabri** [comment]
@jdrossl I think this is already done via the existing siteName replacement mechanics. Please validate.

**Comment by jdrossl** [comment]
@sumerjabri I think you mean during site creating right? If that's the case that doesn't solve the issue because the name would still be fixed after that and pushed to the remote repo

@avasquez614 can you confirm

**Comment by avasquez614** [comment]
@jdrossl @sumerjabri correct, this needs to be dynamically replaced when the file is read. The whole point is that the file can be committed to a remote repository and used on any site

**Comment by jdrossl** [comment]
@avasquez614 @sumerjabri maybe this is a special use case that I'm not aware, but I don't see how this would work if the site name will always be in the URL. If you use the same config with a different site name wouldn't all existing blob files be immediately broken?

**Comment by avasquez614** [comment]
@jdrossl this is mostly for live targets. In cloud, we're already seeing environments with multiple sites that share the same preview blob target but each one has a different live blob target.

**Comment by sumerjabri** [comment]
Must make sure the blob store is RO to all but one site. Related to Duplicate Site mechanics.
